### PR TITLE
hca: Missing Required Field: Insurance policy number or group code.

### PR DESCRIPTION
**Pull request by lihanli**
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/10858

### CODE REVIEW

**Comment by jbalboni** [comment]
@lihanli Needs a package.json update, I think.

File: dist/10-10EZ-schema.json [hunk 1]
```diff
@@ -272,18 +272,18 @@
       "properties": {
         "first": {
           "type": "string",
-          "pattern": "^.*\\S.*",
           "minLength": 1,
-          "maxLength": 30
+          "maxLength": 30,
+          "pattern": "^.*\\S.*"
```
Comment (rianfowler) [inline dist/10-10EZ-schema.json]: Does this order matter?

File: src/schemas/10-10EZ/schema.js [hunk 2]
```diff
@@ -42,12 +42,11 @@ let schema = {
       type: 'object',
       oneOf: countryStateProperites,
       properties: {
-        street: {
+        street: Object.assign({
```
Comment (rianfowler) [inline src/schemas/10-10EZ/schema.js]: This would probably look better using the spread operator.

File: dist/10-10EZ-schema.json (hunk 1 above)
Comment (lihanli) [reply dist/10-10EZ-schema.json]: order of object keys?

Comment (rianfowler) [reply dist/10-10EZ-schema.json]: Yeah was just wondering if this was changed on purpose or just some undone edits.

Comment (lihanli) [reply dist/10-10EZ-schema.json]: the order shouldn't matter, order of object keys isn't guaranteed in js

**Pull request merged**